optimum/bettertransformer/models/__init__.py

```diff
@@ -18,7 +18,7 @@
     BartEncoderLayerBetterTransformer,
     BertLayerBetterTransformer,
     DistilBertLayerBetterTransformer,
-    ViltLayerBetterTransformer,
```

Comment:
This has been removed by mistake, could you please revert that change? 🙏

Reply:
Done